Source/Core/DolphinQt/Settings/AdvancedPane.cpp

```diff
@@ -264,6 +311,30 @@ void AdvancedPane::Update()
     return tr("%1% (%2 MHz)").arg(QString::number(percent), QString::number(clock));
   }());
 
+  QFont vi_bf = font();
+  vi_bf.setBold(Config::GetActiveLayerForConfig(Config::MAIN_VI_FREQUENCY_ENABLE) !=
+                Config::LayerType::Base);
+  m_vi_rate_override_checkbox->setFont(vi_bf);
+  m_vi_rate_override_checkbox->setChecked(enable_vi_rate_override_widgets);
+
+  m_vi_rate_override_slider->setEnabled(enable_vi_rate_override_widgets);
+  m_vi_rate_override_slider_label->setEnabled(enable_vi_rate_override_widgets);
+
+  {
+    const QSignalBlocker blocker(m_vi_rate_override_slider);
+    m_vi_rate_override_slider->setValue(
+        static_cast<int>(std::round(Config::Get(Config::MAIN_VI_FREQUENCY) * 100.f)));
+  }
+
+  m_vi_rate_override_slider_label->setText([] {
+    int percent = static_cast<int>(std::round(Config::Get(Config::MAIN_VI_FREQUENCY) * 100.f));
+    float vps =
+        static_cast<float>(Core::System::GetInstance().GetVideoInterface().GetTargetRefreshRate());
+    if (vps == 0.0f || !Config::Get(Config::MAIN_VI_FREQUENCY_ENABLE))
+      vps = 59.94f * Config::Get(Config::MAIN_VI_FREQUENCY);
```

Comment:
"59.94" isn't always the original VI frequency. I'd just not have that in the UI if it's not feasible to make it accurate.

Reply:
59.94 is only used if no game has been launched yet, and it becomes an accurate value when a game is launched.

Reply:
I think it shouldn't show any VPS in that case, instead of guessing that it will be 59.94.

Reply:
For what it's worth, the CPU overclock setting "guesses" in a similar way (it doesn't know whether you're going to run a GC game or a Wii game).

Reply:
Yeah, I don't mind it guessing as we already do it.

Reply:
I suppose it's fine for now.

Reply:
I'm not against changing it, too.  It's just so inconsequential I wasn't even thinking about it.